File: ios/Interactable/PhysicsArea.m
//
//  PhysicsArea.m
//  Interactable
//
//  Created by Tal Kol on 2/11/17.
//  Copyright © 2017 Wix. All rights reserved.
//

#import "PhysicsArea.h"

@implementation PhysicsArea

- (instancetype)init
{
    if ((self = [super init]))
    {
        self.minPoint = CGPointMake(-CGFLOAT_MAX, -CGFLOAT_MAX);
        self.maxPoint = CGPointMake(CGFLOAT_MAX, CGFLOAT_MAX);
    }
    return self;
}

- (instancetype)initWithMinPoint:(CGPoint)minPoint maxPoint:(CGPoint)maxPoint
{
    if ((self = [super init]))
    {
        self.minPoint = minPoint;
        self.maxPoint = maxPoint;
    }
    return self;
}

- (id)copyWithZone:(__unused NSZone *)zone
{
    return self;
}

@end

File: android/interactable/src/main/java/com/wix/interactable/physics/PhysicsArea.java
package com.wix.interactable.physics;

import android.graphics.Point;

/**
 * Created by zachik on 12/02/2017.
 */

public class PhysicsArea {
     Point minPoint;
     Point maxPoint;

    public PhysicsArea(Point minPoint, Point maxPoint) {
        this.minPoint = minPoint;
        this.maxPoint = maxPoint;
    }
}
